appveyor.yml

```diff
@@ -13,7 +13,7 @@ configuration:
 before_build:
   - git submodule update --init --recursive
   - python get_steinberg_sdk.py
-  - cmake -G "Visual Studio 14" Smexoscope
+  - cmake -G "Visual Studio 14 2015 Win64" .
```

Comment:
Actually here looks like a reasonable way to do it.

http://stackoverflow.com/questions/28350214/how-to-build-x86-and-or-x64-on-windows-from-command-line-with-cmake

You just shadow build in two separate directories.